examples/incident_management/update_incidents.py

```diff
@@ -99,14 +99,17 @@ def cli(attack_surface_rules, status, severity):
     }
 
     # Run bulk update
-    update_results = client.incidents.update(
-        incident_ids=incidents_to_update, update_data=update_data
-    )
-    status_code = update_results.response.status_code
-    if status_code >= 400:
-        raise UnexpectedResponseError(f"Unexpected status code {status_code}.")
-
-    print(f"Update {'succeeded' if update_results.data else 'failed'}.")
+    for incident_id in incidents_to_update:
+        update_results = client.incidents.update(
+            incident_id=incident_id, update_data=update_data
+        )
+        status_code = update_results.response.status_code
+        if status_code >= 400:
```

Comment:
@ben-expanse didn't catch this before, but shouldn't `>= 300` be unexpected?

Reply:
I always go back and forth on whether or not to include 300 status codes as errors. I think sometimes using proxies can return a 300 level code and still return data. But I think using 200-299 here is also resonable